packages/oidc/src/get-vercel-oidc-token.ts

```diff
@@ -21,8 +27,23 @@ import { getContext } from './get-context';
  * });
  * ```
  */
-export async function getVercelOidcToken(): Promise<string> {
-  return getVercelOidcTokenSync();
+export async function getVercelOidcToken(refresh = false): Promise<string> {
```

Comment:
For consumers of this function, I think we'd want the default to be `true`. IIUC, the dream is to just call  `getVercelOidcToken()` and always have a usable OIDC token (so long as you're logged in to development), without having to think about it.

Actually, what do you think about removing the `refresh` parameter entirely?

Reply:
sounds good to me